src/resources/views/crud/columns/password.blade.php

```diff
@@ -0,0 +1,32 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['value'] = str_repeat("*", strlen($column['value']));
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['limit'] = $column['limit'] ?? 6;
+    $column['prefix'] = $column['prefix'] ?? '';
+    $column['suffix'] = $column['suffix'] ?? '';
+    $column['text'] = $column['default'] ?? '-';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['prefix'].Str::limit($column['value'], $column['limit'], '').$column['suffix'];
+    }
+@endphp
+
+<span>
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
+        @if($column['escaped'])
+            {{ $column['text'] }}
+        @else
+            {!! $column['text'] !!}
+        @endif
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_end')
+</span>

```

Comment:
The less custom code we have, the better. So maybe we can do something like this?

```suggestion
@php
    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
    $column['value'] = str_repeat("*", strlen($column['value']));
    $column['limit'] = $column['limit'] ?? 6;
@endphp

@include('crud::columns.text')

```